### PR TITLE
completed TF-LSTM model build.

### PR DESCRIPTION
**Pull request by WillNovus**
I have completed the task of developing a tensorflow model with LSTM using the GloVe embeddings. 
I scraped over 1000 tweets with Nitter, cleaned and preprocessed it before training the TF model. 
I used GPU T4 x 2 for compute.
The link to the experimental notebook is attached below.
[https://www.kaggle.com/code/willgabriel/x-crypto-tweets-sentiment-analysis/edit#Load-GloVe-Embeddings](url)

The script was compiled with the functional steps alongside necessary imports. 
An extra script was provided for  the case of a sample prediction,
The model, vectorizer and datasets developed are also provided in the committed folder.

The LSTM model had a performance accuracy of 76% on a test set of 206 0bservations.

### CODE REVIEW

**Comment by PeterOche** [comment]
can you please ensure that all newly installed dependencies are listed in the requirements.txt file

**Comment by WillNovus** [comment]
I have added a file that contains the dependencies.

**Review by PeterOche — APPROVED** [review]
lgtm
